modules/doc/content/newsletter/2021_11.md

```diff
@@ -67,3 +90,8 @@ If any issues with the new packages are experienced, please open a new
   version variable.
 
 ## Bug Fixes and Minor Enhancements
+
+- Missing user documentation was added for initial conditions, time steppers, kernels, mesh generators,
+  postprocessors, vector postprocessors and nodal kernels.
+- Function-based initial conditions were added for the conserved variable formulation of
+  the fully compressible Navier Stokes equations
```

Comment:
does this apply to both finite volume and finite element?

Reply:
I think so. They both use material properties, and they both can use initialization of conserved through primitive values.
I think NSIC existed before I added to it too